ios/RNZoomUs.m

```diff
@@ -243,6 +247,19 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
   meetingPromiseReject = nil;
 }
 
+#pragma mark - Screen share functionality
+
+- (void)onSinkMeetingActiveShare:(NSUInteger)userID {
+  MobileRTCMeetingService *ms = [[MobileRTC sharedRTC] getMeetingService];
+  if (ms) {
+    if (userID == 0) {
+      [self sendEventWithName:@"MeetingEvent" event:@"screenShareStopped"];
+    } else if (userID == [ms myselfUserID]){
+      [self sendEventWithName:@"MeetingEvent" event:@"screenShareStarted"];
+    }
+  }
```

Comment:
Is there any issue if the we try to stop a meeting that hasn't been started? Or does the `if(ms)` protect us from this?

Reply:
This should only get triggered if there is a meeting underway. However, I am not entirely confident of that, so I am checking to make sure that we can access the meeting service before trying to do anything with it.

Reply:
That'd make sense. I'm guessing you probably wouldn't be presented an option to trigger sharing if it wasn't ready. Or I least let's hope not 😄